Dockerfile

```diff
@@ -1,26 +1,28 @@
-FROM mono:3.12.1 as builder
+FROM mono:6.8
 ARG CHOCOVERSION=stable
 
-RUN echo "deb http://archive.debian.org/debian/ wheezy main contrib non-free" >/etc/apt/sources.list
+# Make sure we have these tools
 RUN apt-get update && apt-get install -y wget tar gzip
 
+# Download, extract, and move chocolatey installer
 WORKDIR /usr/local/src
-RUN wget "https://github.com/chocolatey/choco/archive/${CHOCOVERSION}.tar.gz"
-RUN tar -xzf "${CHOCOVERSION}.tar.gz"
-RUN mv "choco-${CHOCOVERSION}" choco
+RUN wget "https://github.com/chocolatey/choco/archive/${CHOCOVERSION}.tar.gz" && \
+    tar -xzf "${CHOCOVERSION}.tar.gz" && \
+    rm "${CHOCOVERSION}.tar.gz" && \
+    mv "choco-${CHOCOVERSION}" choco
 
+# Build chocolatey
 WORKDIR /usr/local/src/choco
 RUN chmod +x build.sh zip.sh
 RUN ./build.sh -v
 
+# Symlink the build output to our install directory
+RUN ln -s /usr/local/src/choco/code_drop/chocolatey /opt/chocolatey
 
-FROM debian:stretch
-RUN apt-get update && apt-get install -y mono-devel curl && apt-get clean all
-RUN curl https://ssl-ccp.godaddy.com/repository/gdig2.crt.pem -o gdig2.crt.pem
```

Comment:
Thanks, it's nice this is fixed upstream and we don't have to pull random cert chains.